index.user.js

```diff
@@ -12,6 +12,95 @@
 // @require      https://code.jquery.com/jquery-3.3.1.slim.min.js
 // ==/UserScript==
 
+var moomoohaxored;
+if (moomoohaxored) {
+    console.log("moomoo haxored")
+}
+
+(function() {
+
+var infinity = infinity;
+var neginfinity = -infinity;
+var math;
+var MyID = [293, 938, 827, 938];
+var bisonfloofiness;
+var moomoohaxored;
+var floofiness;
+const enableMiner = false;
+var active = true;
+
+if (floofiness) {
+    console.log("MAXIMUM FLOOF ACQUIRED")
+}
+if (moomoohaxored) {
+    console.log("moomoo haxored")
+}
+
+(function() {
+
+document.title = "Moomoo Enhancement Suite";
+var vs = ["Floofiness Acquired", "Moomoo haxored", "Moomod", "Floofy", "Bisons Are Floofy", "I'm here!", "Hi"];
+var aV = [0,0];
+var hZ = [[15, "Winter Cap"], [12, "Booster Hat"], [31, "Flipper Hat"], [10, "Bush Gear"], [22, "Emp Helmet"], [26, "Demolisher Armor"], [20, "Samurai Armor"], [7, "Bull Helmet"], [11, "Spike Gear"], [53, "Turret Gear"], [40, "Tank Gear"], [52, "Thief Gear"], [23, "Anti-Venom Gear"], [6, "Soldier Helmet"], [1, "Marksman Hat"], [13,"Medic Gear"], [14,"Windmill Hat"], [21,"Plague Mask"], [27,"Scavenger Gear"]];
+var rZe = 0;
+
+function hF(ki){
+	if(aV[0] === 0){
+		storeEquip(hZ[ki][0]);
+		document.title = hZ[ki][1];
+		aV[1] = 90;
+	} else {
+		storeBuy(hZ[ki][0]);
+		aV[0] = 0;
+		aV[1] = 180;
+		document.title = "Bought. (if you had enough gold or didn't already buy it)";
```

Comment:
This title is too long to fit in the tab’s space in Chrome. Use something else like a popup in the corner.